index.html

```diff
@@ -16,10 +16,11 @@
         <nav class="displayMobileNav">
           <a href="index.html" class="navBarColor logo">Welcome</a>
           <div class="hamburgerMenu">
-            <i class="fa-solid fa-bars"></i>
+            <i id="hamburgerSymbol" class="fa-solid fa-bars"></i>
           </div>
         </nav>
-        <nav class="displayDesktop">
+        <nav id="popUpMenuMobile" class="displayDesktop">
+          <i id="closeMenuSymbol" class="closeMenuSymbol fa-duotone fa-x"></i>
```

Comment:
- [ ] Great job so far, this x button appears on the desktop size devices, kindly fix it. 

![Screen Shot 2022-09-05 at 4 31 49 PM](https://user-images.githubusercontent.com/59416136/188461336-1d099dfd-19ef-4b28-b1cc-db4a0fe36259.png)